assembly/vm-mock/env.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ *  Set mock for smartcontract call. When called multiple time, mocked values will be added in the mock stack.
+ *
+ * @param value - Value returned by the mocked function
+ */
```

Comment:
Can you add some documentation explaining that this function expects an array of returned values?
Maybe an example?

It might also be interesting to explain that the values are returned sequentially.

Reply:
ok for the example.. but this does not expect an array. just a value to be used as return of the mock

Reply:
> just a value to be used as return of the mock

which is an array 😄